### ISSUE TITLE
the performance of the released pertrained model

### ISSUE THREAD

**Issue by hcy96** [flyywh/Recurrent-Multi-Frame-Deraining#4]
Hi, I test the released pre-trained model (w gan )on the RainSynAll100 testset.

The performance is lower (22.85, 0.85) than the result in the paper (25.14,0.91).

Could you please give me some suggestions to reproduce the result in the paper?

**Comment by flyywh** [comment]
We use wo_gan version for the metric evaluation and w_gan for the real-video testing. Thanks!

**Comment by mapleTan1113** [comment]
老哥问我想问数据集的存放路径要怎么搞，代码里训练时用了两个数据集是怎么回事
Recurrent_multi_frame_deraining_wo_gan/train_video_rain.py，line228左右的
        data_loader = utils.create_data_loader(train_dataset, opts, "train")
        data_haze_loader = utils.create_data_loader(train_haze_dataset, opts, "train")
